src/test/java/com/hocs/test/pages/to/TOProgressCase.java

```diff
@@ -107,6 +121,9 @@ public void completeTheTOStageSoThatCaseMovesToTargetStage(String stageToComplet
                     case "STOP LIST":
                         moveCaseFromTriageToStopList();
                         break;
+                    case "CCH RETURNS":
+                        moveCaseFromTriageToCCH();
```

Comment:
The first line of this switch case should set the businessArea value to "BF" or "HMPO", then we wont need the if statement below which could slow down the scenarios.